src/main/data_structure/queue.py

```
@@ -0,0 +1,40 @@
+class Queue:
+
+    # Constructor to initiate queue
+    def __init__(self, queue=None):
+        if queue is None:
+            queue = []
+        self.queue = queue
+
+    # push to append an item into queue
+    def push(self, value):
+        self.queue.append(value)
+
+    # front to return starting element from queue
+    def front(self):
+        if Queue.empty(self) is False:
+            return self.queue[0]
+        else:
+            return "An empty Queue."
+
+    # pop to remove front element from queue
+    def pop(self):
+        if Queue.empty(self) is False:
+            self.queue.pop(0)
+        else:
+            return "An empty Queue."
```

raise an exception instead of returning An empty Queue
https://www.w3schools.com/python/gloss_python_raise.asp#:~:text=Raise%20an%20exception,exception%2C%20use%20the%20raise%20keyword